src/rewriter/CMakeLists.txt

```diff
@@ -50,13 +50,19 @@ libcvc5_add_sources(GENERATED
   ${REWRITE_OUTPUT_FILES}
 )
 
-# If the generated files differ from the source files, they should be copied to
+# If the generated header differs from the source header, it should be copied to
```

Comment:
Should `cvc5_proof_rule.cpp` be removed from the block below? (line 56)

Reply:
No. `cvc5_proof_rule.cpp` was added to `LIBCVC5_GEN_SRCS` instead. I moved the comment there to explain why the generated `cvc5_proof_rule.h` is not tracked.

Reply:
No. `cvc5_proof_rule.cpp` was added to `LIBCVC5_GEN_SRCS` instead. I moved the comment there to explain why the generated `cvc5_proof_rule.h` is not tracked.